theseus/_version.py

```diff
@@ -0,0 +1,13 @@
+import warnings
+
+from torch import __version__ as _torch_version
+from semantic_version import Version
+
+if Version(_torch_version) < Version("2.0.0"):
+    warnings.warn(
+        "Using torch < 2.0 is deprecated and support will be discontinued "
```

Comment:
will be discontinued -> is discontinued?

Reply:
Changed the message here 103219c2df6db8c924fcc16649faa0e5380b5402.